test/samlp.logout.session_store.tests.js

```diff
@@ -800,6 +886,66 @@ describe('samlp logout with Session Participants - Session Provider', function (
         });
       });
     });
+
+    describe('SP initiated - 1 Session Participant with POST binding', function () {
```

Comment:
It looks like we have no existing test coverage for `options.protocolBinding` handling - how do you feel about adding that to check we haven't introduced a regression?